### PR TITLE
peer reviewed schema changes... and more

### PR DESCRIPTION
**Pull request by joedevivo**
This is an open PR to facilitate the conversation about this schema. There is work to be done here.

### CODE REVIEW

File: priv/multi_backend.schema [hunk 1]
```diff
@@ -62,17 +62,3 @@
   end
  end
 }.
-
```
Comment (joedevivo) [inline priv/multi_backend.schema:65]: Moved this into the riak_kv.schema next to it's friends.

File: priv/riak_kv.schema [hunk 2]
```diff
@@ -100,60 +98,51 @@
...
   {default, "15s"},
-  {datatype, {duration, ms}}
+  {datatype, {duration, ms}},
+  {level, advanced}
 ]}.
 
 %% @doc The directory where AAE hash trees are stored.
 {mapping, "anti_entropy.data_dir", "riak_kv.anti_entropy_data_dir", [
-  {default, "{{platform_data_dir}}/anti_entropy"}
+  {default, "{{platform_data_dir}}/anti_entropy"},
```
Comment (seancribbs) [inline priv/riak_kv.schema:109]: `{datatype, directory}`

File: priv/riak_kv.schema [hunk 3]
```diff
@@ -164,237 +153,220 @@
...
+ fun(Conf) ->
+   case cuttlefish:conf_get("object.format", Conf) of
+       0 -> v0;
+       1 -> v1;
+       _ -> throw(error)
+   end
+ end
+}.
+
 %% @doc md_cache_size controls how large the md cache is allowed to be
```
Comment (seancribbs) [inline priv/riak_kv.schema]: What's 'md'? How does the user know what it is?

**Comment by joedevivo** [comment]
We talked about this to death today, should be good to merge... right @seancribbs? right?!

**Comment by seancribbs** [comment]
:see_no_evil: :hear_no_evil: :speak_no_evil:

**Comment by angrycub** [comment]
:+1:

**Comment by seancribbs** [comment]
1+

**Comment by joedevivo** [comment]
The only monkey emoticon not here, is the one that's now off my back!

**Pull request merged**